packages/flutter/lib/src/widgets/animated_child_switcher.dart

```diff
@@ -62,8 +109,45 @@ class AnimatedChildSwitcher extends StatefulWidget {
   /// The duration over which to perform the cross fade using [FadeTransition].
   final Duration duration;
 
+  /// The builder function used to generate custom transitions for
+  /// [AnimatedChildSwitcher].
```

Comment:
talk about the default, and maybe something about the contract here?

Reply:
Done.